### PR TITLE
Header User Dropdown

### PR DESCRIPTION
**Pull request by grigoriu-stefan-iulian**
Screenshot from 2020-03-29 21-49-46

### CODE REVIEW

File: app/components/Layout/Header.js [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
     </div>
     <HeaderNotifications />
     <div className="app-my-account">
-      <span>Welcome, Luca</span>
-      <DownOutlined />
+      <Dropdown overlay={menuItems}>
+        <div>
+          <div className="app-my-account__user">
+            <UserOutlined />
+            <span className="app-my-account__user-name"> Luca Luca Luca Luca Luca</span>
```
Comment (pancudaniel7) [inline app/components/Layout/Header.js]: Question:
- we will adopt combined naming strategy ? "app-my-account__user-name" ?

File: app/containers/Settings/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        </div>
+        <div className="box-content">
+          Settings Page
+        </div>
+      </div>
+    </MainContainerInner>
+  </MainContainerOuter>
+);
+
+export default Settings;
```
Comment (pancudaniel7) [inline app/containers/Settings/index.js]: 👍

File: app/components/Layout/constants.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Menu } from 'antd';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { default as Logout } from '../Logout/index';
+
+export const menuItems = (
```
Comment (pancudaniel7) [inline app/components/Layout/constants.js]: Questions:
- This is a constant ?
- If so the naming convention should not look like this: "MENU_ITEMS" ?

File: app/components/Layout/Header.js (hunk 1 above)
Comment (grigoriu-stefan-iulian) [reply app/components/Layout/Header.js]: Andrei said it's OK to use it but we can drop it if we don't want BEM in css

File: app/components/Layout/constants.js (hunk 3 above)
Comment (grigoriu-stefan-iulian) [reply app/components/Layout/constants.js]: Yes, I will use the convention in the project

File: app/components/Layout/Header.js (hunk 1 above)
Comment (pancudaniel7) [reply app/components/Layout/Header.js]: As you decide

File: app/components/Layout/constants.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { Menu } from 'antd';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { default as Logout } from '../Logout/index';
+
+export const MENU_ITEMS = (
```
Comment (pancudaniel7) [inline app/components/Layout/constants.js]: 👍

**Pull request merged**